mailer.rst

```diff
@@ -1120,6 +1120,39 @@ a specific address, instead of the *real* address:
                 ],
             ],
         ]);
+        
+Functional Test
+~~~~~~~~~~~~~~~~~~
+
+To functionally test that an email was sent, and even assert the email content or headers, you can use the built in assertions.
+
+.. configuration-block::
+
+    .. code-block:: php
+    	
+        // tests/Controller/MailControllerTest.php
```

Comment:
It is not "configuration-block". So that should be removed. We can also use the shortcut to create code block. 

```suggestion
To functionally test that an email was sent, and even assert the email content or headers, you can use the built in assertions.::
    	
    // tests/Controller/MailControllerTest.php
```

Note that we need to fix the indentation on all remaining rows.